server/tournaments/index.ts

```diff
@@ -1211,6 +1226,34 @@ const commands: {basic: TourCommands, creation: TourCommands, moderation: TourCo
 			}
 			this.sendReply(`|html|<div class='infobox infobox-limited'>This tournament includes:<br />${tournament.getCustomRules()}</div>`);
 		},
+		export(tournament) {
+			if (this.room.lastTournament && this.room.lastTournament.type === 'tree') {
```

Comment:
You initialize `room#lastTournament` as `{}`, so check isn't right. I'd probably initialize it as `null`, so if it's present at all it'll have the correct data.

Reply:
OK thanks.